src/tests/functional/shared_test_classes/src/base/utils/generate_inputs.cpp

```diff
@@ -270,43 +270,41 @@ ov::runtime::Tensor generate(const std::shared_ptr<ov::op::v0::FakeQuantize>& no
     }
 }
 
-ov::runtime::Tensor generate(const std::shared_ptr<ov::op::v0::PSROIPooling>& node,
+ov::Tensor generate(const std::shared_ptr<ov::op::v0::PSROIPooling>& node,
                              size_t port,
                              const ov::element::Type& elemType,
                              const ov::Shape& targetShape) {
     const auto &inputShape = node->get_input_shape(0);
     if (port == 1) {
-        ov::runtime::Tensor tensor = ov::test::utils::create_and_fill_tensor(elemType, targetShape);
-        LayerTestsDefinitions::PSROIPoolingLayerTest::fillROITensor(tensor.data<float>(),
-                                                                    tensor.get_size() / 5,
-                                                                    inputShape[0],
-                                                                    inputShape[2],
-                                                                    inputShape[3],
-                                                                    node->get_group_size(),
-                                                                    node->get_spatial_scale(),
-                                                                    node->get_spatial_bins_x(),
-                                                                    node->get_spatial_bins_y(),
-                                                                    node->get_mode());
+        auto tensor = ov::Tensor(elemType, targetShape);
+        ov::test::utils::fill_psroi(tensor,
+                                    inputShape[0],
+                                    inputShape[2],
+                                    inputShape[3],
+                                    node->get_group_size(),
+                                    node->get_spatial_scale(),
+                                    node->get_spatial_bins_x(),
+                                    node->get_spatial_bins_y(),
+                                    node->get_mode());
```

Comment:
@olpipi @iefode please, review this part in detail - it's mostly copy/paste, perhaps need to format it to some common style

Reply:
Just suggestion, maybe keep roi number in the argument of `ov::test::utils::fill_psroi` will be more readable and flexible?

Reply:
@riverlijunjie good suggestion! But because `fill_psroi` becomes hidden function for the file, I see no reason to do so until it will required. Also I did it in order to decrease number of arguments